dataland-frontend/tests/e2e/specs/user-api-key/GenerateApiKey.ts

```diff
@@ -121,18 +68,13 @@ describe("As a user I expect my api key will be generated correctly", () => {
     cy.get("h1").should("contain.text", "Create new API Key");
   }
 
-  it("check Api Key functionalities", () => {
+  it("Check Api Key functionalities", () => {
     cy.ensureLoggedIn();
+    cy.intercept("GET", "**/api-keys/getApiKeyMetaInfoForUser*").as("getApiKeyMetaInfoForUser");
+    cy.intercept("GET", "**/api-keys/generateApiKey*").as("generateApiKey");
```

Comment:
Warum hast du den `generateApiKey` intercept hierhin ausgelagert. Ich bin da an sich kein Fan von. Das macht die Funktion in der darauf gewartet wird abhängiger vom Aufrufkontext.

Reply:
FYI, intercept hat eine `times` property, die festlegt wie oft ein intercept ausgeführt wird.

Reply:
Nice Danke für den Tipp mit `times`.

Ich habe das Intercept hier rausgelagert, weil es ursprünglich ein paar mal wiederholt wurde im File hier.
Weil die Funktionen hier nur zum "Gruppieren" von Teilen des Tests dienen, fand ich es ok.

Wenn es dir absolut nicht gefällt kann ich es aber auch wieder rückgängig machen.

Alternativ könnten wir auch die Gruppierung mithilfe von zwei Funktionen hier auch einfach auflösen.    Ursprünglich waren das mal vier Funktionen, da hat das noch viel Sinn gemacht.
Jetzt wo es nur noch zwei sind, könnte man drüber streiten.

Reply:
Jo ich komm schon klar. Wird ja nur hier verwendet.